presto-product-tests/src/main/java/io/prestosql/tests/hive/TestCsv.java

```diff
@@ -174,4 +175,67 @@ private static void assertSelect(String query, String tableName)
                 .hasColumns(expected.getColumnTypes())
                 .containsOnly(expectedRows);
     }
+
+    @Test(groups = STORAGE_FORMATS)
+    public void testWriteCsvTableWithMultiCharProperties()
```

Comment:
move below `testReadCsvTableWithMultiCharProperties`, as a person reasing this class should see `testReadCsvTableWithMultiCharProperties` first, we logically depend on results of that test

Reply:
Makes sense.